src/devices/ovrheadset/CMakeLists.txt

```diff
@@ -13,6 +13,12 @@ yarp_prepare_plugin(ovrheadset
 if(NOT SKIP_ovrheadset)
   set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
+  if(YARP_EXPERIMENTAL_API)
+    option(OVR_DEVICE_EXERIMENTAL "experimental version of OvrDevice" OFF)
+  endif()
+  if(OVR_DEVICE_EXERIMENTAL)
+    add_definitions(-DOVR_EXPERIMENTAL)
```

Comment:
Same as the other comment, I'd use the same name for the option and for the definition